wagtail_content_import/pickers/microsoft/utils.py

```diff
@@ -0,0 +1,39 @@
+from django.template.loader import render_to_string
+from django.utils.safestring import mark_safe
+
+from .. import Picker
+from ...parsers import get_docx_parser
+
+
+def parse_document(document):
+    parser = get_docx_parser()
+    return parser(document).parse()
+
+
+class MicrosoftPicker(Picker):
+    name = "microsoft"
+    verbose_name = "SharePoint"
```

Comment:
is "SharePoint" accurate here? AFAIK the same picker works for OneDrive and SharePoint.

Reply:
Hmm - this is only used in the menu button text. I'm not sure which is better, OneDrive/SharePoint is quite long, but you're right, it works for both.

Microsoft, maybe? Or OneDrive as it is more commonly used?

Reply:
I vote OneDrive

Reply:
Great, thanks, I've changed it over in latest commit.